Scripts/datahandling/resultdata.py

```diff
@@ -115,7 +115,9 @@ def print_matrices(self,
                     sheet_name=f"{description}_{key}")
         # Create text file
         stacked_matrices = pandas.concat(
-            {f"{description}_{key}": df.stack() for key, df in data.items()},
-            names=["purpose", "orig", "dest"])
+            {key: df.stack() for key, df in data.items()},
+            names=["mode", "orig", "dest"])
```

Comment:
I played a bit with `pandas.concat` and realized that you can achieve the same result like this:

```suggestion
            {(description, key): df.stack() for key, df in data.items()},
            names=["purpose", "mode", "orig", "dest"])
```